RxSwiftWorkshop/Shared/IATAService/IATAService.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Created by Maciej Oczko on 01.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import Marshal
+import Keys
+
+protocol IATACallable {
+    func airport(for code: String) -> Observable<Airport>
+    func airline(for code: String) -> Observable<Airline>
+}
+
+struct IATAService: IATACallable {
+    struct Constants {
+        static let baseURL = "http://iatacodes.org/api/v6/"
```

Comment:
Maybe we could provide a URL object here and avoid the fatalError/implicit unwrap at the usage site?